pkg/rpc/settings.go

```diff
@@ -50,15 +50,15 @@ var useDialback = settings.RegisterBoolSetting(
 
 var enableRPCCompression = envutil.EnvOrDefaultBool("COCKROACH_ENABLE_RPC_COMPRESSION", true)
 
-func getWindowSize(name string, c ConnectionClass, defaultSize int) int32 {
+func getWindowSize(ctx context.Context, name string, c ConnectionClass, defaultSize int) int32 {
```

Comment:
Since we don't yet allow reconfiguration and compute the same value on every call maybe call out why we do this in a comment? Otherwise someone changing a caller may decide to cache the value indefinitely by mistake.